### DISCUSSION TITLE
is there a way of telling VT that I don't have AC?

### DISCUSSION THREAD

**Discussion by doubledrat** [jmcollin78/versatile_thermostat#761] [Q&A] [answered]
as title.  I'm having to hide lots of ac settings because I don't have it.  is there a simple way of letting it know not to bother with AC entities?

**Comment by jmcollin78** [comment]
Hello, yes just don't check the 'AC mode'. Seems too easy.. Can you tell me more ? Give me your VTherm attributes please (cf. https://github.com/jmcollin78/versatile_thermostat/blob/main/documentation/en/reference.md#custom-attributes)

**Comment by doubledrat** [comment]
I've not seen an ac mode option.  Where is it?  I've been through all the
configure options again and am not seeing it.

I'm afraid the only thermostat I can see in states is my real one.  I can
see plenty of options for yours, like

number.central_configuration_preset_boost_ac_temp

But nothing that resembles your example.

Sorry to be such a noob!

**Reply by doubledrat** [reply, marked as answer]
OK.  I've added my first real radiator valve (a sonoff trv) and on that I can see an "AC mode".  before I was looking in central configuration and that didn't have the same options.

**Reply by jmcollin78** [reply]
Is ti ok now ?

**Comment by doubledrat** [comment]
I think so,  I was concerned I would have 16 lots of ac settings to hide,
but they've only appeared on the central configuration
